src/riddl/FooBarSameDomain/FooBar.conf

```diff
@@ -0,0 +1,22 @@
+# options for hugo-translator
+command = hugo
+common = {
+  show-times = true
+  verbose = true
+  quiet = false
+  dry-run = false
+  show-warnings = true
+  show-missing-warnings = false
+  show-style-warnings = false
+}
+hugo {
+  input-file = "examples/src/riddl/FooBarSameDomain/FooBar.riddl"
+  output-dir = "examples/target/translator/FooBarSameDomain"
+  project-name = "FooBar"
+  erase-output = true
+  base-url = "https://riddl.tech"
+  source-url = "https://github.com/reactific/riddl"
+  edit-path = "blob/main/examples/src/riddl/FooBarSameDomain"
+  site-logo-path = "images/FooBar.png"
+}
+
```

Comment:
could you please also add the below snippet at the end so we can `from ... validate` too? Thanks
```
validate {
   input-file = "examples/src/riddl/FooBarSameDomain/FooBar.riddl"
}```

Reply:
Done